app/experimenter/nimbus-ui/src/components/AppLayout/index.tsx

```diff
@@ -3,16 +3,33 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 
 type AppLayoutProps = {
+  testid?: string;
   children: React.ReactNode;
 };
 
 // TODO: Depending on the main page ("directory view") design, we may want two
 // AppLayouts - one with a header/footer, and another with a nav sidebar.
 
-export const AppLayout = ({ children }: AppLayoutProps) => {
-  return <main data-testid="main">{children}</main>;
+export const AppLayout = ({ children, testid = "main" }: AppLayoutProps) => {
+  return (
+    <Container
+      fluid
+      as="main"
+      className="h-100"
+      data-testid={testid}
+    >
+      <Row className="h-100">
+        <Col className="m-5">
```

Comment:
This could be added on EXP-462 (I can add it as a suggestion there) but adding `ml-auto mr-auto col-md-10 col-lg-8` makes this look nice if you need to repush anything else before merge :)

Reply:
Thanks! I'm still kind of wrapping my head around the responsive layout classes in Bootstrap, so m-5 was a total guess :)